src/data/css-quiz.ts

```diff
@@ -1197,6 +1197,17 @@ const cssQuiz = [
     Explanation:
       "We can use the :last-of-type selector to target an element of a specific type within a container.",
     Link: "https://developer.mozilla.org/en-US/docs/Web/CSS/:last-of-type/"
+  },
+   {
+    Question:
+      "In order to display a groove kind of border, which property should we use?",
+    Answer: "border-style",
+    Distractor1: "border-bottom-color",
+    Distractor2: "border-style",
```

Comment:
distractors are for wrong answers only

```suggestion
    Distractor2: "border-length",
```